core/src/main/java/tc/oc/pgm/command/util/CommandKeys.java

```diff
@@ -16,4 +17,7 @@ public class CommandKeys {
 
   public static final CloudKey<MapPoolType> POOL_TYPE =
       CloudKey.of("_pgm_pool_type_param_", new TypeToken<MapPoolType>() {});
+
+  public static final CloudKey<Phase> PHASE =
```

Comment:
I dont believe this to be necessary

It exists for others because a param can determine what another param allows/suggests, like a setting key determines what setting values show up in a `/set <key> [value]`, you dont have that need here